agent/src/main/scala/za/co/absa/atum/agent/model/MeasuresBuilder.scala

```diff
@@ -17,16 +17,16 @@
 package za.co.absa.atum.agent.model
 
 import za.co.absa.atum.agent.exception.MeasureException
-import za.co.absa.atum.agent.model.Measure._
+import za.co.absa.atum.agent.model.AtumMeasure._
 import za.co.absa.atum.model.dto
```

Comment:
Why not importing `MeasureDTO` straight instead of importing the whole package while using only module.

Reply:
Thanks, I didn't notice it. Changed the import as per your suggestion.